CAIPs/caip-ipld_log.md

```diff
@@ -0,0 +1,105 @@
+---
+caip: <to be assigned>
+title: IPLD Timestamp Proof
+author: Zach Ferland <@zachferland>, Joel Thorstensson (@oed)
+discussions-to: <>
+status: Draft
+type: Standard
+created: 2022-11-10
+updated: 2022-11-10
+requires: <>
+---
+
+## Simple Summary 
+
+Authenticated IPLD Append-Only Logs provide the best of IPLD and an authenticated immutable underlying log with the ability to model mutable databases and data structures on top. 
+
+## Abstract
+
+This specification combines IPLD and DagJWS (with optional use of CAIP-X DagJWS CACAO), with a minimal log based data structure. A log based data structure being a simple to define subset of a general IPLD DAG. 
+
+## Motivation
+
+Append-only logs are a frequently used underlying immutable data structure in distributed systems to improve data integrity, consistency, performance, history, etc. Open distributed systems use hash linked lists/logs, to allow others to verify the integrity of any data. IPLD is a natural way to define an immutable append-only log. Combined with DagJWS and "CAIP-X DagJWS CACAO" it allows authenticated writes to these logs using blockchain accounts. Providing a common data base layer for users and applications besides more expensive on-chain data or centralized and siloed databases. 
+
+A minimally defined log structure allows a base level of interoperability while allowing diverse implementations of mutable databases and data structures on top. Base levels of interoperability could include log transport, update syncing, consensus, etc.
+
+## Specification
+
+Logs are made up of events. A genesis event being the origination of a new log and typically used to reference or name a log. Every additional “update” appended is a data event. This specification provides minimal definition of an IPLD log, parameters in both the headers and body can be added by specific implementations or protocols. Formats and types are described using [IPLD schema language](https://ipld.io/docs/schemas/).
+
+Being a CAIP, with the optional use of "CAIP-X DagJWS CACAO", keys and references to signers are expected to be did:pkh or DIDs. General specifications would not necessarily rely on did:pkh or DIDs.
+
+All events are DAGJWS payloads encoded in IPLD using the [DAG-JOSE codec](https://ipld.io/specs/codecs/dag-jose/spec/).
+
+### Genesis Event
+
+A log is initialized with a genesis event. The CID of this event can be used to reference or name this log. 
+
+```tsx
+type GenesisHeader struct {
+  controllers [String]
+}
+type GenesisEvent struct {
+  header GenesisHeader
+  data Any | &Any
```

Comment:
not sure how to define ANY or CID of ANY in ipld schema language

Reply:
```suggestion
  data Any | Link
```